package.json

```diff
@@ -5,6 +5,7 @@
   "license": "MIT",
   "scripts": {
     "dev": "vite",
```

Comment:
Why did you decide to use `vite` instead of ie. `cra`?

Reply:
I'm a little bit more familiarized with vite lately.

Also, with some custom configuration CRA requires either eject or useage of library like `react-app-rewired`, so you end up complex configuration.